### PR TITLE
feat(JitsiConference): Separate conference name validation errors.

### PR DESCRIPTION
**Pull request by Romick2005**
Separate thrown errors:  "no conference name passed" and "it contains invalid characters like capital letters".

### CODE REVIEW

File: JitsiConference.js [hunk 1]
```diff
@@ -476,7 +481,7 @@ JitsiConference.prototype._init = function(options = {}) {
             userName: config.statisticsDisplayName ? config.statisticsDisplayName : this.myUserId(),
             confID: config.confID || `${this.connection.options.hosts.domain}/${this.options.name}`,
             siteID: config.siteID,
-            roomName: this.options.name,
```
Comment (saghul) [inline JitsiConference.js:479]: Why are you changing this?

**Comment by saghul** [comment]
Since the errors are indistinguishable (no code or other attribute) why is this change useful?

File: JitsiConference.js (hunk 1 above)
Comment (Romick2005) [reply JitsiConference.js:479]: My bad that should be roomName. Initially I used roomName = options.name, but then I notice that it is mention as conference name in some places and room name on others so renamed all occurrence and this code was corrupted (

**Comment by Romick2005** [comment]
So you can close it if it is not important.

**Pull request closed without merging**